src/screens/Account/TokenAccountsList.js

```diff
@@ -63,36 +71,104 @@ const Card = ({ children }: { children: any }) => (
   <View style={styles.card}>{children}</View>
 );
 
-const Header = ({ tokenAccounts }: { tokenAccounts: TokenAccount[] }) => (
-  <View style={styles.header}>
-    <LText
-      fontWeight="500"
-      style={{ color: colors.darkBlue, fontSize: 16 }}
-    >{`Tokens (${tokenAccounts.length})`}</LText>
-  </View>
-);
-
-const Footer = () => (
-  <View style={styles.footer}>
-    <Icon color={colors.live} size={26} name="plus" />
-    <View style={styles.footerText}>
-      <LText style={{ fontSize: 16 }}>
-        <Trans i18nKey="account.tokens.howTo">
-          <LText semiBold>text</LText>
-          <LText semiBold>text</LText>
-        </Trans>
-      </LText>
-    </View>
-  </View>
-);
-
 const TokenAccountsList = ({
   tokenAccounts,
   onAccountPress,
+  navigation,
+  accountId,
 }: {
   tokenAccounts: TokenAccount[],
   onAccountPress: TokenAccount => *,
+  navigation: *,
+  accountId: string,
 }) => {
+  const [isCollapsed, setCollapsed] = useState(true);
+  const renderHeader = useCallback(
```

Comment:
post PR feedback: a dedicated Header component would be preferred in this kind of big comopnents.